docs/2.guide/4.recipes/3.custom-usefetch.md

```diff
@@ -0,0 +1,94 @@
+---
+navigation.title: 'Custom useFetch'
+title: Custom useFetch in Nuxt
+description: How to create a custom fetcher for calling your external API in Nuxt 3.
+---
+
+When working with Nuxt, most of the time your are making the frontend and fetching an API made with another language (ex: PHP with Laravel).
+
+When we created the [`useFetch`](/docs/api/composables/use-fetch) composable and [`$fetch`](/docs/api/utils/dollarfetch) utility function, we made them immutable on purpose to avoid any side-effects. This is important to keep a consistent behaviour through your application and avoid any modules to break your API calls, for example.
```

Comment:
```suggestion
When we created the [`useFetch`](/docs/api/composables/use-fetch) composable and [`$fetch`](/docs/api/utils/dollarfetch) utility function, we made them immutable on purpose to avoid any side effects. This is important to keep a consistent behavior throughout your application and avoid any modules to break your API calls, for example.
```